ioc_module.js

```diff
@@ -1,4 +1,3 @@
-'use strict';
```

Comment:
Wieso ist das hier gelöscht?

In `.js` Dateien muss das drin bleiben.

```suggestion
'use strict';
```